bundles/org.openhab.core.io.transport.mqtt/src/main/java/org/eclipse/smarthome/io/transport/mqtt/MqttBrokerConnection.java

```diff
@@ -114,7 +112,6 @@
      * The callback will interact with the {@link AbstractReconnectStrategy} as well as inform registered
      * {@link MqttConnectionObserver}s.
      */
-    @NonNullByDefault
```

Comment:
Why did you remove that?

Reply:
My Eclipse told me, that it was unnecessary as the outer class has it already:
`Nullness default is redundant with a default specified for the enclosing type MqttBrokerConnection`

Reply:
I think I added it because SAT complains about a missing annotation. Maybe the SAT check is wrong.

Reply:
I think, SAT is wrong.
But I checked SAT and removed all warnings.